catalogue/webapp/jest.e2e.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  preset: 'jest-playwright-preset',
+  testRegex: ['.e2e.test.(js|ts)'],
```

Comment:
I'd much rather this in a folder, but happy to put it to the team.
i.e. /test/e2e/

Reply:
And could we remove the `js` file? We should be working in `ts` only.